### PR TITLE
Puma 6 support

### PR DESCRIPTION
**Pull request by severinkaelin**
Addresses #42

I am currently in the process of final testing of the changes. Feedback is welcome.

### CODE REVIEW

**Comment by harmjanblok** [comment]
Please include change in puma-metrics.gemspec#L25 in commit 1674984 as well.

**Comment by severinkaelin** [comment]
> Please include change in puma-metrics.gemspec#L25 in commit [1674984](https://github.com/harmjanblok/puma-metrics/commit/167498453a84c4170ed8808d6e706a2dc05181c1) as well.

@harmjanblok Thanks for the hint. I've just adjusted it. 
Heads-up: The tests are currently not passing. I haven't had a chance to look at the errors in detail yet.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Changes:
+- Require puma 6.0 or newer
```
Comment (severinkaelin) [inline CHANGELOG.md:6]: @harmjanblok Judging by [this comment](https://github.com/harmjanblok/puma-metrics/pull/43#issuecomment-1279130206), I assume that's how you want to proceed, right?

**Comment by kevingriffin** [comment]
We use this, and we'd love to see it merged. Is there any way I can help?

**Comment by severinkaelin** [comment]
> We use this, and we'd love to see it merged. Is there any way I can help?

@kevingriffin Glad to hear that. Thanks for your feedback. I am happy to make further contributions as needed and as  time allows. @harmjanblok How do you want to proceed? What else do we need etc.?

**Review by harmjanblok — APPROVED** [review]

**Comment by harmjanblok** [comment]
Looks good to me, I'll squash the commits and merge it in a moment. I'll release a new version soon.

**Pull request merged**